app/controllers/application.js

```diff
@@ -56,6 +55,8 @@ export default Ember.Controller.extend({
     let url = window.location.href;
     return !containsAny(url, [
       "request_purpose",
+      "search_goods",
```

Comment:
need to remove this "search_goods" from here

Reply:
Also @patrixr 's recent changes for the sidebar fix are not included in this PR.

Reply:
@abhinav098 Those are in another PR.